Silicon/CommonSocPkg/Include/Library/GpioLib.h

```diff
@@ -0,0 +1,352 @@
+/** @file
+  The platform GPIO library header.
+
+  Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _GPIO_LIBRARY_H_
+#define _GPIO_LIBRARY_H_
+
+#include <Base.h>
+#include <Uefi/UefiBaseType.h>
+#include <Library/IoLib.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/GpioPlatformLib.h>
+
+/**
+  Set GPIO PAD DW 0 and 1 registers for GPIO communities
+
+  @param[in] Gpio_Pin_Num    GPIO Pin Number to configure
+  @param[in] Gpio_Conf_Data  GPIO_CONF_PAD_INIT data array
+                             for each GPIO communities.
+**/
+VOID
+GpioPadConfigTable (
+  IN UINT32              Gpio_Pin_Num,

```

Comment:
All the functions in the header file are the library API to be exposed to others.
so please add EFIAPI for those function in both header file and C file.

Reply:
Sure, I will make sure x64 gcc build passes from my next patch.